src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionProvider.cs

```diff
@@ -62,51 +59,45 @@ protected override async Task AddCompletionItemsAsync(CompletionContext completi
         /// Multiple aliases might live under same namespace.
         /// Key is the namespace of the symbol, value is the name of the symbol.
         /// </summary>
-        private MultiDictionary<string, string> GetAliasTypeDictionary(
+        private static MultiDictionary<string, string> GetAliasTypeDictionary(
             Document document,
             SyntaxContext syntaxContext,
             CancellationToken cancellationToken)
         {
             var syntaxFactsService = document.GetRequiredLanguageService<ISyntaxFactsService>();
             var dictionary = new MultiDictionary<string, string>(syntaxFactsService.StringComparer);
 
-            var nodeToCheck = syntaxContext.LeftToken.Parent;
-            if (nodeToCheck == null)
-            {
-                return dictionary;
-            }
-
-            // In case the caret is at the beginning of the file, take the root node.
-            var aliasDeclarations = GetAliasDeclarationNodes(nodeToCheck);
-            foreach (var aliasNode in aliasDeclarations)
+            foreach (var scope in syntaxContext.SemanticModel.GetImportScopes(syntaxContext.Position, cancellationToken))
             {
-                var symbol = syntaxContext.SemanticModel.GetDeclaredSymbol(aliasNode, cancellationToken);
-                if (symbol is IAliasSymbol { Target: ITypeSymbol { TypeKind: not TypeKind.Error } target })
+                foreach (var symbol in scope.Aliases)
```

Comment:
nit: def an interesting use of the word 'SyntaxContext' when it's now giving access to semanticmodels and symbols.  but i don't hate it. it's the contextual info (including sematnics) at a particular syntactic location.  so i think it fits.